src/client/hud.cpp

```diff
@@ -371,7 +371,7 @@ void Hud::drawLuaElements(const v3s16 &camera_offset)
 										 (e->number >> 0)  & 0xFF);
 				std::wstring text = unescape_translate(utf8_to_wide(e->text));
 				core::dimension2d<u32> textsize = textfont->getDimension(text.c_str());
-#ifdef __ANDROID__
+#ifdef HAVE_TOUCHSCREENGUI
```

Comment:
this seems to be more of an android thing(?)

Reply:
Yeah, I'll revert that.